corehq/apps/app_manager/models.py

```diff
@@ -823,8 +824,12 @@ def wrapped_xform(self):
     def validate_form(self):
         vc = self.validation_cache
         if vc is None:
+            # formtranslate requires all attributes to be valid xpaths, but
```

Comment:
Hmm. I did verify that this will fail, but it doesn't seem to fail with form comments: https://github.com/dimagi/Vellum/blob/master/tests/static/comments/comment-test.xml. Although it probably only looks for certain xpath without looking at the namespace. 

@wspride do you know know what the jar looks for when validating an xform?